web/src/composebox_typeahead.ts

```diff
@@ -596,6 +596,9 @@ export function filter_and_sort_mentions(
 }
 
 export function get_pm_people(query: string): (UserGroupPillData | UserPillData)[] {
+    if (query === "" && $("#private_message_recipient").siblings().length > 0) {
+        return [];
+    }
```

Comment:
I think you should be able to remove this now

Reply:
I wrote this check to make sure typeahead appears only when there is no recipient with empty string. I though this was the intended design.(?) Subsequent empty strings would open typeahead on click

Reply:
Can you explain what would happen if this code wasn't here? Like what could the user do to end up in an undesirable state?

Reply:
If a recipient user pill is already present, clicking inside the recipient box (possibly to add another user) will open up typeahead since we have `helponemptrystring` set to true. I though it was only for empty recipient box(?)
>we should try immediately opening the addressee typeahead when there is no recipient selected

Reply:
I think that might be desirable behavior actually. Can you ask on CZO to see if people prefer the typeahead opening or not?